sources/lib/Serializer/Normalizer/FlexibleEntityNormalizer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/*
+ * This file is part of the PommProject/PommBundle package.
+ *
+ * (c) 2018 Grégoire HUBERT <hubert.greg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PommProject\PommBundle\Serializer\Normalizer;
+
+use PommProject\ModelManager\Model\FlexibleEntity\FlexibleEntityInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+class FlexibleEntityNormalizer implements NormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($object, $format = null, array $context = array())
+    {
+        return $object->extract();
```

Comment:
I would build an `stdClass` and forward it to a the normalizer again. (By implementing `NormalizerAwareInterface`)